internal_packages/keybase/lib/keybase-search.cjsx

```diff
@@ -0,0 +1,68 @@
+{Utils, React} = require 'nylas-exports'
+PGPKeyStore = require './pgp-key-store'
+KeybaseUser = require '../lib/keybase-user'
+kb = require './keybase'
+_ = require 'underscore'
+
+module.exports =
+class KeybaseSearch extends React.Component
+  @displayName: 'KeybaseSearch'
+
+  constructor: (props) ->
+    super(props)
+    @state = {
+      query: ""
+      results: []
+      loading: false
+    }
+
+    @debouncedSearch =  _.debounce(@_search, 200)
+
+  _search: ->
+    if @state.query != "" and @state.loading == false
+      @setState({loading: true})
+      kb.autocomplete(@state.query, (profiles) =>
+        if profiles?
```

Comment:
Just from a minor styling standpoint, it'd be sort of nice to unify these code paths like this:

```
@setState({
   results: (profiles || []).map (profile) -> {keybase_user: profile},
   loading: false,
})


```